src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,110 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
  * реалізації ArrayList (default capacity, newCapacity...)</p>
  */
 public class ArrayList<T> implements List<T> {
+    private static final int CAPACITY = 10;
+    private int size = 0;
+    private T [] arrayList;
+
+    public ArrayList() {
+        arrayList = (T[]) new Object[CAPACITY];
+    }
+
+    public ArrayList(int size) {
```

Comment:
did you test if this constructor work? 
it's not neccessary for this task, so i suggest we remove it

Reply:
this constructor is optional for tests, but in general it should be in just the arraylist class